rust/automerge-test/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "automerge-test"
-version = "0.2.0"
+version = "0.2.1"
 edition = "2021"
```

Comment:
Or should this be 0.3.0?

Reply:
Er, I think it's fine. Probably?

Reply:
Based on the precedent of de5af2f I believe it should be `0.3.0`